core/common/src/main/java/alluxio/security/LoginUser.java

```diff
@@ -97,7 +97,7 @@ private static User login(Configuration conf) throws IOException {
       }
       return userSet.iterator().next();
     } catch (LoginException e) {
-      throw new IOException("Failed to login", e);
+      throw new IOException("Failed to login: " + e);
```

Comment:
could you also include `e` as the cause so that it's stack trace and cause is available in case someone does want more details about the IOException?